README.md

```diff
@@ -46,16 +49,19 @@ There are two types of management connectivity for NOS VMs: _pass-through_ and _
 _Pass-through management_ interfaces allows the use of the assigned management IP within the NOS VM, management traffic is transparently passed through to the VM, and the NOS configuration can accurately reflect the management IP. However, it is no longer possible to send or receive traffic directly in the vrnetlab container (e.g. for installing additional packages within the container), other than to pre-defined exceptions, such as the QEMU serial port on TCP port 5000.
 
 NOSes defaulting to _pass-through_ management interfaces are:
-- All vJunos routers
+
+* All vJunos routers
 
 In case of _host-forwarded_ management interfaces, certain ports are forwarded to the NOS VM IP, which is always 10.0.0.15/24. The management gateway in this case is 10.0.0.2/24, and outgoing traffic is NATed to the container management IP. This management interface connection mode does not allow for traffic such as LLDP to pass through the management interface.
 
 NOSes defaulting to _host-forwarded_ management interfaces are:
-- Every NOS not listed as pass-through management
+
+* None so far, we are gathering feedback on this, and will update this list as feedback is received. Please contact us in [Discord](https://discord.gg/vAyddtaEV9) or open up an issue here if you have found any issues when trying the passthrough mode.
```

Comment:
I think this note should be moved to pass-through/transparent management management.

Reply:
hah, I think I edited the wrong line

Reply:
hopefully done right in [8d977ba](https://github.com/hellt/vrnetlab/pull/287/commits/8d977ba365bc6db73cd1cb1c3506617948ac699b)